.github/actions/gitops-release/action.yaml

```diff
@@ -109,9 +109,9 @@ runs:
       working-directory: gitops
       run: |
         if [[ "${{ inputs.canary-master-values-path }}" == "release" ]]; then
-          FULL_PATH="apps/${{ inputs.gitops-name }}/release/chart/values.yaml"
+          FULL_PATH="bootstrap/apps/${{ inputs.gitops-name }}/values/release.yaml"
         else
-          FULL_PATH="apps/${{ inputs.gitops-name }}/branches/${{ inputs.canary-master-values-path }}/chart/values.yaml"
+          FULL_PATH="bootstrap/apps/${{ inputs.gitops-name }}/values/development.yaml"
```

Comment:
This still needs to be data driven. Using `${{ inputs.canary-master-values-path }}`?